### PR TITLE
Specialist briefs can be open for 1 or two weeks

### PR DESCRIPTION
**Pull request by Wynndow**
For [this](https://www.pivotaltracker.com/story/show/123706751) story on Pivotal.

This PR adds the question for the requirements length to the specialist brief and bumps the version to 2.1.0.

### CODE REVIEW

File: frameworks/digital-outcomes-and-specialists/questions/briefs/questionAndAnswerSessionDetailsSpecialists.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+id: questionAndAnswerSessionDetails
+name: Question and answer session details
+question: Question and answer session details
+question_advice: |
+  <br>
```
Comment (idavidmcdonald) [inline frameworks/digital-outcomes-and-specialists/questions/briefs/questionAndAnswerSessionDetailsSpecialists.yml]: Do we want this `<br>` tag in here? Not sure if intentional or not? Had a search in the repo and looks like we use this very infrequently so wondered if there is a different way to do this or not...

Comment (Wynndow) [inline frameworks/digital-outcomes-and-specialists/questions/briefs/questionAndAnswerSessionDetailsSpecialists.yml]: Yeah, that's a good question. Probably not...but worth checking with @ralph-hawkins

Comment (Wynndow) [inline frameworks/digital-outcomes-and-specialists/questions/briefs/questionAndAnswerSessionDetailsSpecialists.yml]: Have had a chat with Ralph, it's there to make the optional flag on the rendered page look less like a heading for the text. He's agreed to remove it and he'll add it to the design technical debt.

**Comment by idavidmcdonald** [comment]
👍

**Pull request merged**